config/environment.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(environment) {
     modulePrefix: 'hn-reader',
     environment: environment,
     baseURL: '/',
-    locationType: 'auto',
```

Comment:
Hm, I heard about the crash, but I am pretty sure it is not caused by this. Do you remember why you ended up with this fix?